src/network/networkprotocol.h

```diff
@@ -1027,20 +1027,20 @@ enum NetProtoCompressionMode {
 	NETPROTO_COMPRESSION_NONE = 0,
 };
 
-const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
+static const char *accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
```

Comment:
When I build, I get spammed with
```
/opt/minetest-vanilla/src/network/networkprotocol.h:1030:20: warning: ‘accessDeniedStrings’ defined but not used [-Wunused-variable]
 1030 | static const char *accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
``` 

though I see it's used here:
https://github.com/minetest/minetest/blob/2500eabcf20ea650f4a7bf0903a9d5dfc2bcf87f/src/network/clientpackethandler.cpp#L202-L225

Reply:
Not sure why it didn't happen locally but I changed it back :shrug: